package.json

```diff
@@ -47,5 +47,8 @@
     "**/@wdio/utils": "5.12.1",
     "**/webdriver": "git+https://github.com/react-native-windows/webdriver.git",
     "**/webdriverio": "5.12.1"
+  },
+  "beachball": {
+    "gitTags": false
```

Comment:
Is this one needed?  This package is never published.

Reply:
Yep, this one is by design. Beachball does the following to generate package options:

```ts
/**
 * Gets all package level options (default + root options + package options + cli options)
 */
export function getPackageOptions(packagePath: string): PackageOptions {
  const configExplorer = cosmiconfigSync('beachball', { cache: false });
  const searchResults = configExplorer.search(packagePath);
  const defaultOptions = getDefaultOptions();
  const rootOptions = getRootOptions();
  return {
    ...defaultOptions,
    ...rootOptions,
    ...(searchResults && searchResults.config),
    ...getCliOptions(),
  };
}
```

`getRootOptions` will search for configuration set at the root of the repo instead of the local package. This means something like `react-native-codegen` will inherit the option we set here in the root to disable tags. Motivation here it to make new packages not tag by default.